SingularityUI/app/components/requestDetail/header/RequestActionButtons.jsx

```diff
@@ -162,7 +162,7 @@ const RequestActionButtons = ({requestParent, fetchRequest, fetchRequestHistory,
 
   const navigateAwayOnSuccess = (response) => {
     if (response.statusCode === 200) {
-      router.push('/requests');
+      then={fetchRequestAndHistory}
```

Comment:
can move this a few lines down and get rid of the `navigateAwayOnSuccess ` method:
`<RemoveButton requestId={request.id} then={fetchRequestAndHistoryAndActiveTasks}>`

- Don't need the 200 check here since we are staying on the page regardless of the outcome of the api call
- Added the `...AndActiveTasks` portion because they will likely be updated to cleaning after the deletion is requested